handlers/stock.js

```diff
@@ -0,0 +1,178 @@
+const azure = require('azure-storage');
+
+/**
+ * This handler manages the global supply of $chreef $tock.
+ * Each new user is given 1000 $$ once they join the channel.
+ * Use this stock as you will.
+
+ * Currently supports:
+ * Checking balance
+ */
+
+const TABLENAME = 'schreeftable';
+const ENTGEN = azure.TableUtilities.entityGenerator;
+const STARTINGSTOCK = 1000;
+
+module.exports = {
+    name: 'stock',
+    command: 'stock',
+    handler: stock
+};
+
+function stock(channel, message, params) {
+
+    // Wont work until this is populated from whatever is managing creds
+    var placeholderConnectionString;
+
+    // Initialize table service
+    const tableSvc = azure.createTableService(placeholderConnectionString);
+    var splitParams = params.split(" ");
+
+    // Verify the Azure Storage Table exists and create if it doesn't
+    tableSvc.createTableIfNotExists(TABLENAME, function (error, result, response) {
+        if (!error) {
+            // Table exists or created
+        }
+    })
+
+    // Verify all users have an account and create new ones for those that don't
+    for (let member of channel.guild.members.array()) {
+        getStock(tableSvc, member.user.username, function (callback) {
+            if (typeof callback != 'number') {
+                console.log("initializing " + member.user.username);
+                updateUser(tableSvc, member.user.username, STARTINGSTOCK);
+            }
+        })
+    }
+
+    if (splitParams[0] == "help") {
```

Comment:
for strings use single quotes `'` whenever possible

Reply:
I disagree with single quotes over double quotes. 

But also this is an issue that'll just fix itself when ESLINT is thrown down anyway.

Reply:
Moving comment as fasthub put it in a weird spot:

I read some back and forth articles on it and bow to whatever style guide we choose. Converted fully to single at the moment and leaving it that way for now.